### ISSUE TITLE
Rewards page shows only one reward per layer

### ISSUE THREAD

**Issue by dshulyak** [spacemeshos/explorer-frontend#160]
It is not clear if it is by design or not, but there are many rewards per layer, and displaying only one of them will confuse everyone.

**Comment by mephistopheles1010** [comment]
@dshulyak can you please give an example on current devnet209(https://explorer.spacemesh.io)? Because I see as example layer 2075, which have two rewards and you can see it in layers overview(https://explorer.spacemesh.io/layers) and in rewards overview(https://explorer.spacemesh.io/rewards).

**Comment by mephistopheles1010** [comment]
Also while investigating this issue, I found new one. While scrolling rewards overview, sometimes the same rewards are shown two times, it happens because new rewards are added to the database while scrolling have already been started and it is offsetting existing pagination frame. If we need to fix it, please create a new issue.

**Comment by dshulyak** [comment]
@mephistopheles1010 i found in [logs](https://kibana-devnet209.spacemesh.io/goto/e2b06a07e663e550508b5e16944d17ba) that layer 2075 has 21 rewards, one per block. 

can you please check if they are stored in the backend? if they are not this is either a bug in the explorer backend or in go-spacemesh.

**Comment by mephistopheles1010** [comment]
@dshulyak On my local machine, there is only one reward stored at level 2075. And also we have strange behavior on public explorer, after refreshing the reward page sometimes we have different data for rewards for layer. It seems like explorer front-end receiving data from different explorer backends, which are connected to different nodes. And in these cases, we can see that we have different rewards saved on explorer's back-end for different nodes, but all of these rewards is exists in logs you provided  above. 
Thus, I can conclude, that either the node doesn't send all rewards to explorer or explorer doesn't accept all rewards. 
To clarify the last point, I need devnet to be up and running.

**Comment by mephistopheles1010** [comment]
It is problem with node streaming api. Blocked by #2384.

**Comment by mephistopheles1010** [comment]
It is fixed with https://github.com/spacemeshos/go-spacemesh/pull/2948. You can check it now on https://explorer.spacemesh.io/rewards, devnt: devnet-kirill-5. In general it works fine with explorer since 0.0.10 and node since v0.2.7-beta.0. This issue can be closed.

**Comment by avive** [comment]
I'd like to verify this but according to the discovery service metadata devnet-k-5 is using explorer 0.0.9.
